ci/jenkins/jobs/projects.yaml

```diff
@@ -147,6 +147,13 @@
           - ${{sha1}}
           builders:
           - builder-integration
+          properties:
+          - throttle:
+              max-per-node: 1
+              max-total: 0
+              categories: integration
+              option: category
+              enabled: true
```

Comment:
In the future, please consider making this change in a separate PR

Reply:
Ok, got it.

Reply:
Is there any issue caused by the concurrency?

Reply:
for the concurrency change, due to all integration jobs are running in one integration VM, it may have conflicts if both auto integration PR and manual integration PR being trigger at the same time. not sure if @lzhecheng have encountered such failure?

Reply:
Yes, I recently saw it happened. It doen't happen frequently because for most of the time, integration test donesn't fail before.